CHANGELOG.md

```diff
@@ -5,8 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- No changes yet.
+
+## [1.59.0] - 2021-11-10
 ### Added
 - tchannel: add option to accept native tchannel handlers.
```

Comment:
I believe we should mention about tchannel-go package version upgrade. The message is unclear whether this change belongs to the tchannel-go package or internal tchannel transport module.